custom/export_demographics.php

```diff
@@ -1,23 +0,0 @@
-<html>
```

Comment:
ok,
then we should keep this file and the below custom/export_labworks.php file then since it may have value.